dataprocess/dataprocess_test.go

```diff
@@ -0,0 +1,40 @@
+package dataprocess
+
+import (
+	"os"
+	"testing"
+
+	"github.com/edaniels/golog"
+	pc "go.viam.com/rdk/pointcloud"
+	"go.viam.com/test"
+)
+
+func TestWriteBytesToFile(t *testing.T) {
+	_ = golog.NewTestLogger(t)
```

Comment:
I recommend moving this into a t.Run so that people don't need to move it into a t.Run when a second test case is added.